erpnext/controllers/queries.py

```
@@ -615,7 +631,7 @@ def get_blanket_orders(doctype, txt, searchfield, start, page_len, filters):
 
 @frappe.whitelist()
 @frappe.validate_and_sanitize_search_inputs
-def get_income_account(doctype, txt, searchfield, start, page_len, filters):
+def get_income_account(doctype: str, txt: str, searchfield: str, start: int, page_len: int, filters: dict):
```

Ruff (0.15.1)

[warning] Unused function argument: `start`

(ARG001)